{"metadata":{"role":"symbol","title":"MFAServiceError.general(message:)","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"general","kind":"identifier"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"modules":[{"name":"MFA"}],"externalID":"s:3MFA15MFAServiceErrorO7generalyACSS_tcACmF","symbolKind":"case"},"kind":"symbol","schemaVersion":{"minor":3,"patch":0,"major":0},"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError"]]},"abstract":[{"type":"text","text":"General error with custom message."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}],"platforms":["iOS"],"languages":["swift"]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError\/general(message:)"},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/mfa\/mfaserviceerror\/general(message:)"]}],"references":{"doc://com.ibm.verify.ios.sdk/documentation/MFA/MFAServiceError":{"type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError","abstract":[],"role":"symbol","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"MFAServiceError","kind":"identifier"}],"navigatorTitle":[{"text":"MFAServiceError","kind":"identifier"}],"title":"MFAServiceError","url":"\/documentation\/mfa\/mfaserviceerror"},"doc://com.ibm.verify.ios.sdk/documentation/MFA":{"type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA","title":"MFA","url":"\/documentation\/mfa","kind":"symbol","abstract":[],"role":"collection"},"doc://com.ibm.verify.ios.sdk/documentation/MFA/MFAServiceError/general(message:)":{"url":"\/documentation\/mfa\/mfaserviceerror\/general(message:)","title":"MFAServiceError.general(message:)","role":"symbol","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError\/general(message:)","type":"topic","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"kind":"symbol","abstract":[{"type":"text","text":"General error with custom message."}]}}}